Projects/App/Sources/Common/Extension/UIButton+.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  UIButton+.swift
+//  GGiriGGiri
+//
+//  Created by 안상희 on 2022/07/27.
+//  Copyright © 2022 dvHuni. All rights reserved.
+//
+
+import UIKit
+
+extension UIButton {
+    func setBackgroundColor(_ color: UIColor?, for state: UIControl.State) {
+        
+        UIGraphicsBeginImageContext(CGSize(width: 1.0, height: 1.0))
+        guard let context = UIGraphicsGetCurrentContext() else { return }
+        guard let color = color else { return }
```

Comment:
```suggestion
        guard 
            let context = UIGraphicsGetCurrentContext(),
            let color = color
        else { return }
```
위와 같이 한줄에 조건을 표시할 수도 있습니다. 분리하신 이유가 있을까요 ? 😄

Reply:
처음에는 color 조건이 없었는데, 추후 추가하면서 저렇게 되었네요😅 한 줄에 조건을 모두 표시하도록 [변경](https://github.com/mash-up-kr/GGiriGGiri_iOS/pull/72/commits/4bd65e6415eafd9474bdadaf15dafaebd336d43e)했습니다!